apps/mull-ui/src/app/components/contact-row/contact-row.tsx

```diff
@@ -27,14 +29,15 @@ export const ContactRow = ({
   modalButton2Text,
   modalButton2OnClick,
   icon = faEllipsisH,
+  id = '',
 }: ContactRowProps) => {
   // TODO: Replace boolean by the appropriate button option according to query
   const addedMe = false;
   const [modalOpen, setModalOpen] = useState<boolean>(false);
 
   return (
-    <div className="contact-container">
-      <Link to={`/user/${user.id}`}>
+    <div className="contact-container" id={id}>
+      <Link to={ROUTES.USER_BY_ID.replace(':id', user.id + '')}>
```

Comment:
Why not have a constants USER = "/user/" and concatenate the id or have a method that does it instead replacing the string everywhere.

Reply:
In this specific case the :id is at the end. This might not always be the case. Why have a separate `"/user/"` variable when I can express where I want the user to go with the routes we already have defined?